### ISSUE TITLE
Drag and drop example not working, Firefox 133.0.3

### ISSUE THREAD

**Issue by mashdragon** [mdn/content#37458]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API

### What specific section or headline is this issue about?

"Examples", specifically Dragging and dropping files

### What information was incorrect, unhelpful, or incomplete?

The two examples which actually appear to be identical

Dragging and dropping files (Firefox only): https://jsfiddle.net/9C2EF/
Dragging and dropping files (All browsers): [https://jsbin.com/hiqasek/](https://jsbin.com/hiqasek/edit?html,js,output)

These both do not work in the latest version of Firefox on Ubuntu. I drag and drop a file, and the files array has length 0. So I get this:
![image](https://github.com/user-attachments/assets/7cc87265-e232-496e-af40-228c449eb0c3)

In Chromium (hosted on localhost) the example has a file but the file size is 0! 

```
File Count: 1
 File 0:
(object) : <[object File] > test1.txt 0
```

In Chromium on an incognito tab at the actual website [https://jsbin.com/hiqasek/](https://jsbin.com/hiqasek/edit?html,js,output) I get

```
File Count: 0
```

### What did you expect to see?

On both browsers the example should accept a file dragged onto the "DROP FILES HERE" div. The file should be readable. Otherwise, if this is an issue with browsers, I think the documentation should be updated to explain the limitations of this function.

(Also, second minor issue: there is a broken Interoperability anchor link in the blue Note section.)

### Do you have anything more you want to share?

This is using Ubuntu snap versions of Firefox and Chromium, updated to the latest version.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/html_drag_and_drop_api`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/html_drag_and_drop_api/index.md
* Last commit: https://github.com/mdn/content/commit/816cc4d4a5a318a23222946b6981bb92b499aebb
* Document last modified: 2024-10-11T12:51:34.000Z

</details>

**Comment by mashdragon** [comment]
I think I found why this is happening. Firefox won't accept dropped files from any temporary directory. It might be due to the snap sandboxing. I don't know.

**Comment by caugner** [comment]
> Firefox won't accept dropped files from any temporary directory.

Yes, it's plausible that the snap versions of Firefox and Chromium don't have permissions to read from the temporary directory, and it's somewhat interesting if Firefox just ignores the file and Chromium treats the file like an empty file.

I'm not sure if it's worth documenting this, but I'll leave this to others to decide.

**Comment by Josh-Cena** [comment]
I looked around, and I think it's fine to leave it as-is. This behavior is not specified by the spec, so we should leave it intentionally vague too, since browsers are free to choose whatever behavior they prefer. In both cases it suggests the file doesn't exist, and there are many ways in which a file can't be read by the browser by the time this event fires—it's more a user error, not something the application should account for. The application can code as if this case doesn't exist.
